packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -584,3 +610,63 @@ class ImportedServerlessCluster extends ServerlessClusterBase implements IServer
     return this._clusterReadEndpoint;
   }
 }
+
+/**
+ * Properties for ``ServerlessClusterFromSnapshot``
+ */
+export interface ServerlessClusterFromSnapshotProps extends ServerlessClusterBaseProps {
+  /**
+   * The identifier for the DB instance snapshot or DB cluster snapshot to restore from.
+   * You can use either the name or the Amazon Resource Name (ARN) to specify a DB cluster snapshot.
+   * However, you can use only the ARN to specify a DB instance snapshot.
+   */
+  readonly snapshotIdentifier: string;
+
+  /**
+   * The secret attached to the database cluster
+   *
+   * @default - no secret
+   */
+  readonly secret?: secretsmanager.ISecret;
+}
```

Comment:
Can we work with `credentials?: Credentials;` here? Like we do for the `DatabaseInstanceFromSnapshot`.

Reply:
Yes, I adopted to logic from `DatabaseInstanceFromSnapshot`.